src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -1193,6 +1193,55 @@ authProvider.2=com.f.MyAuth2
     only one will be used. Also all servers must have the same plugins defined, otherwise clients using
     the authentication schemes provided by the plugins will have problems connecting to some servers.
     </para>
+
+    <para> <emphasis role="bold">Added in 3.6.0</emphasis>: An alternate abstraction is available for pluggable
+    authentication. It provides additional arguments.
+    </para>
+
+    <programlisting>
+public abstract class ServerAuthenticationProvider implements AuthenticationProvider {
```

Comment:
I'm copying the other parts of the zookeeperProgrammers.xml here. I built the docs and it looks good.